src/main/java/javaClasses/homework_5/davit_balabekyan/transport/Taxi.java

```diff
@@ -0,0 +1,34 @@
+package javaClasses.homework_5.davit_balabekyan.transport;
+
+import javaClasses.homework_5.davit_balabekyan.transport.interfaces.Bookable;
+import javaClasses.homework_5.davit_balabekyan.transport.interfaces.Printer;
+import javaClasses.homework_5.davit_balabekyan.transport.interfaces.Stoppable;
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+@ToString
```

Comment:
Well done! But keep mind  that overusing sometimes can be bad

Reply:
OK)